lib/Parser/CaseInsensitive.cpp

```diff
@@ -579,9 +579,18 @@ END {
         static const int numTransforms = sizeof(transforms) / sizeof(Transform);
         static const Transform lastTransform = transforms[numTransforms - 1];
 
+        static constexpr bool isTransformEntryValid(int offset)
+        {
+            return offset >= numTransforms || ((transforms[offset].delta0 != transforms[offset].delta1 || transforms[offset].delta0 != transforms[offset].delta2 || transforms[offset].delta0 != transforms[offset].delta3) && isTransformEntryValid(offset + 1));
```

Comment:
Break up this line to make it more readable, and add a comment about the static recursive call.

Reply:
Done, and I added checks for two more properties of the table.